tests/Resources/ODS.robot

```diff
@@ -3,3 +3,11 @@ Resource  ./Page/LoginPage.robot
 Resource  ./Page/ODH/ODHDashboard/ODHDashboard.resource
 Resource  ./Page/ODH/JupyterHub/ODHJupyterhub.resource 
 Resource  ./Page/ODH/Prometheus/Prometheus.resource
+
+
+*** Keywords ***
+Container Image Should Be
```

Comment:
I think this keyword should be in tests/Resources/OCP.resource and probably be renamed to `Verify Container Image` to have a similar naming than the existing keywords in that file

Reply:
okay
Thanks